### PR TITLE
Enhancement style base

### PR DESCRIPTION
**Pull request by alsturm**
add: input and select base style
add: showcase refactoring
add: toolbox and draw style

### CODE REVIEW

File: src/modules/iconSelect/components/IconSelect.js [hunk 1]
```diff
@@ -62,6 +66,8 @@ export class IconSelect extends MvuElement {
 				return { ...model, color: data };
```
Comment (taulinger) [inline src/modules/iconSelect/components/IconSelect.js:66]: test for signal type `Update_Color` is missing

also for setter
```

	set value(value) {
		this.signal(Update_Value, value);
	}
```

File: src/modules/toolbox/components/drawToolContent/DrawToolContent.js [hunk 2]
```diff
@@ -67,6 +69,10 @@ export class DrawToolContent extends AbstractToolContent {
 				return { ...model, tools: data };
 			case Update_FileSaveResult:
 				return { ...model, fileSaveResult: data.fileSaveResult };
+			case Update_CollapsedInfo:
+				return { ...model, collapsedInfo: data };
+			case Update_CollapsedStyle:
+				return { ...model, collapsedStyle: data };
```
Comment (taulinger) [inline src/modules/toolbox/components/drawToolContent/DrawToolContent.js:75]: no tests for both signal types

**Review by taulinger — COMMENTED** [review]
Very nice proceedings!!
We just have to add some missing tests...

File: src/modules/iconSelect/components/IconSelect.js [hunk 3]
```diff
@@ -33,22 +33,26 @@ export class IconSelect extends MvuElement {
 			icons: [],
 			color: null,
 			value: null,
-			isCollapsed: true
+			isCollapsed: true,
+			portrait: false
```
Comment (taulinger) [inline src/modules/iconSelect/components/IconSelect.js:37]: Please extend the test for default values of `isCollapsed` and `portrait`  -> "contains default values in the mode..."

File: src/modules/iconSelect/components/IconSelect.js (hunk 1 above)
Comment (taulinger) [reply src/modules/iconSelect/components/IconSelect.js:66]: Missing yet

File: src/modules/toolbox/components/drawToolContent/DrawToolContent.js [hunk 4]
```diff
@@ -67,6 +70,10 @@ export class DrawToolContent extends AbstractToolContent {
...
 		}
 	}
```
Comment (taulinger) [inline src/modules/toolbox/components/drawToolContent/DrawToolContent.js:79]: Also for this MvuElement we should add a test that verifies its initial model...

File: src/modules/toolbox/components/drawToolContent/DrawToolContent.js [hunk 5]
```diff
@@ -12,6 +12,8 @@ import { AssetSourceType, getAssetSource, hexToRgb } from '../../../map/componen
 const Update = 'update';
 const Update_Tools = 'update_tools';
 const Update_FileSaveResult = 'update_fileSaveResult';
```
Comment (taulinger) [inline src/modules/toolbox/components/drawToolContent/DrawToolContent.js:14]: signal for this type is never called. Can we remove it?

**Review by taulinger — COMMENTED** [review]
Almost done 🤓

File: src/modules/toolbox/components/drawToolContent/DrawToolContent.js (hunk 5 above)
Comment (thiloSchlemmer) [reply src/modules/toolbox/components/drawToolContent/DrawToolContent.js:14]: No... a observer was mistakingly removed with the last commit. I'll fix that.

**Review by taulinger — APPROVED** [review]
Yeah, finally done 😁

**Pull request merged**